packages/graph/hash_graph/lib/graph/src/types/schema/mod.rs

```diff
@@ -38,21 +44,31 @@ impl fmt::Display for ValidationError {
             Self::MissingRequiredProperty(uri) => {
                 write!(
                     fmt,
-                    "The schema has marked the \"{uri}\" property as required, but it wasn't \
+                    "the schema has marked the \"{uri}\" property as required, but it wasn't \
                      defined in the `\"properties\"` object"
                 )
             }
+            Self::BaseUriMismatch {
+                base_uri,
+                versioned_uri,
+            } => {
+                write!(
+                    fmt,
+                    "the base URI ({base_uri}) of the referenced versioned URI ({versioned_uri}) \
+                     is not matching"
```

Comment:
I don't think this is clear. I think we should have something more like:

"expected base URI `{base_uri}` differed from the base URI of `{versioned_uri}`"

There's probably a better way to phrase it but I think the current one is quite confusing.

Reply:
Could we add some delimiters around the URI still?

Reply:
Applied in 50b816777f1da61569fdb47ed5dff349ceed9f7f